test/openssl/utils.rb

```diff
@@ -115,6 +115,10 @@ def libressl?(major = nil, minor = nil, fix = nil)
     return false unless version
     !major || (version.map(&:to_i) <=> [major, minor, fix]) >= 0
   end
+
+  def aws_lc?(major = nil, minor = nil, fix = nil)
+    return true if OpenSSL::OPENSSL_VERSION.include?("AWS-LC")
+  end
```

Comment:
This can be simplified:

```suggestion
  def aws_lc?
    OpenSSL::OPENSSL_VERSION.include?("AWS-LC")
  end
```

Reply:
Fixed! Thanks, really appreciate any tips with Ruby as it's not my forte.